src/components/Primary-Button/Button.constants.ts

```diff
@@ -0,0 +1,58 @@
+import type { ButtonVariant, ButtonSize, ButtonShape } from './Button.types';
+
+export const BUTTON_VARIANTS = {
+  primary: 'primary',
+  secondary: 'secondary',
+} as const;
+
+export const BUTTON_SIZES = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+export const BUTTON_SHAPES = {
+  rounded: 'rounded',
+  square: 'square',
+  pill: 'pill',
+} as const;
+
+export const SIZE_CONFIGS = {
+  small: { height: 32, minWidth: 64, paddingScale: [0.75, 2], fontSizeRem: 0.8125 },
+  medium: { height: 40, minWidth: 72, paddingScale: [1, 2.5], fontSizeRem: 0.875 },
+  large: { height: 48, minWidth: 80, paddingScale: [1.25, 3], fontSizeRem: 1 },
+} as const;
+
+export const SHAPE_CONFIGS = {
```

Comment:
SHAPE_CONFIGS contains raw numeric border-radius values. The project styling rules require avoiding hard-coded px/hex in components and prefer theme tokens (theme.shape.borderRadius or spacing-based tokens). Storing raw numbers in constants increases risk of inconsistent radii across components and makes theme changes harder.

Suggestion:
- Either store shape as semantic tokens (e.g. 'rounded', 'pill', 'square') and map these to theme values in the styles file using theme (so constants stay theme-agnostic), or provide the shape multipliers and compute borderRadius in StyledButton using theme.spacing(...) or theme.shape.borderRadius * multiplier.